private.sh

```diff
@@ -148,7 +148,8 @@ pg::_gh_migrate() {
 # Sources a file if it exists, skips if not
 pg::_srcIfExist() {
   if [[ -f "${1:?}" ]];  then
-    pg::log "Including $(pg::relative_path "$1" "$_PGEM_LOC")"
+    # pg::relative_path is slow, so we need to guard this call
+    if "${_PGEM_DEBUG:-false}"; then pg::log "Including $(pg::relative_path "$1" "$_PGEM_LOC")"; fi;
```

Comment:
Looks like a regression introduced by 27337b16; on my Mac:

```shell
$ time for i in {1..100}; do pg::relative_path functions.sh "$PWD/.." > /dev/null; done

real	0m3.719s
user	0m2.240s
sys	0m0.949s

$ time for i in {1..100}; do pg::_dispPath functions.sh > /dev/null; done

real	0m0.530s
user	0m0.318s
sys	0m0.399s
```

Even before that, though, this wasn't worth the overhead since users rarely run with `_PGEM_DEBUG`.